UM/Settings/ContainerQuery.py

```diff
@@ -73,6 +73,8 @@ def execute(self, candidates: Optional[List[Any]] = None) -> None:
             if isinstance(value, str):
                 if ("*" or "|") in value:
                     key_filter = lambda candidate, key = key, value = value: self._matchRegExp(candidate, key, value)
+                elif value[0] == "[" and value[-1] == "]":
```

Comment:
Didn't putting | in the filter work? That's what the two lines above this seem to imply.

Reply:
Fixed this by moving the if-block to be the first one. Made a few changes to the matching pattern and added a unit test for this.